Assignment _5/main.cpp

```diff
@@ -0,0 +1,81 @@
+#include <iostream>
+#include <fstream>
+
+//#include './library.h"
+
+const std::string CONFIG = "C:/Users/Tim/ClionProjects/Assignment_2_File_Generator/config.txt";
+const std::string EDIT = "C:/Users/Tim/ClionProjects/Assignment_2_File_Generator/edit.txt";
+
+void init();
+void edit();
+
+
+int main(int argc, char *argv[]) {
+
+    if (static_cast<std::string>(argv[1]) == "init"){
+        init();
+
+    }
+
+    else if (static_cast<std::string>(argv[1]) == "edit"){
+        edit();
+
+    }
+return 0;
+}
+void init(){
```

Comment:
Add a parameter to pass the the path. It will be needed when you turn this into a library.
`void init(std::string config_file_path)`